<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1231</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ejercicio 1</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>50</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese nombre de la clase:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxUV">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>190</y>
        <width>101</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCargar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>490</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Cargar</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBoxCalificacion">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>130</y>
        <width>141</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMateria">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>271</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese el Nombre del estudiante:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Seleccione El tipo de estudiante: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGuardar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>540</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxTipoEstudiante">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNombre">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxEdad">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>170</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese U.V de la clase:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMostrarPromedios">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>440</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Mostrar Promedios</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>110</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese la nota de la clase:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Digite la edad: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAgregarEstudiante">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>390</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar Estudiante</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ejercicio 2</string>
     </attribute>
     <widget class="QPushButton" name="anadir">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Añadir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="remover">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="guardar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modeloEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="anospinBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2030</number>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="kilometrajedoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000000000.990000009536743</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="capacidadCargaDoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <double>100000.990000000005239</double>
      </property>
     </widget>
     <widget class="QSpinBox" name="viajesSpinBox">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>190</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>250</y>
        <width>331</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="tipocomboBox">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Auto</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Camion</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Modelo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Año:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Kilometraje</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Capacidad de Carga</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>160</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Vaijes</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Ejercicio 3</string>
     </attribute>
     <widget class="QLineEdit" name="nombrelineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="edadspinBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="salariodoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maximum">
       <double>200000.989999999990687</double>
      </property>
     </widget>
     <widget class="QComboBox" name="TipocomboBox2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Empleado Fijo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Empleado Contratado</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="horasspinBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>200</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="empleadosTableWidget">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>80</y>
        <width>511</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>440</y>
        <width>371</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>260</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Cargar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Edad</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Salario</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Horas trabajdas</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1269</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
